internal/helpers.go

```diff
@@ -276,10 +276,10 @@ func ReplaceFileContent(filPath string, oldContent string, newContent string) er
 // UpdateTerraformS3BackendForK8sAddress during the installation process, Terraform must reach port-forwarded resources
 // to be able to communicate with the services. When Kubefirst finish the installation, and Terraform needs to
 // communicate with the services, it must use the internal Kubernetes addresses.
-func UpdateTerraformS3BackendForK8sAddress(k1Dir string) error {
+func UpdateTerraformS3BackendForK8sAddress(k1Dir string, gitopsRepoName string) error {
```

Comment:
Same comment as above, to simplify reading, you can merge these declarations to make the function signature shorter:

```suggestion
func UpdateTerraformS3BackendForK8sAddress(k1Dir, gitopsRepoName string) error {
```

Since they're both strings, you don't need to say `foo string, bar string` but instead `foo, bar string`.